src/domains/citizen-auth-app/01_network_itn.tf

```diff
@@ -61,3 +61,48 @@ resource "azurerm_private_endpoint" "staging_function_lollipop_itn_sites" {
 
   tags = var.tags
 }
+
+
+## fn-fast-login-itn
+
+resource "azurerm_private_endpoint" "function_fast_login_itn_sites" {
```

Comment:
I think it would be better to add a dependency to the function

Reply:
What do you mean with dependency? Could be enough a comment that refers the function declaration?